src/service.py

```diff
@@ -31,6 +31,7 @@ def main():
 
     logger.info('Starting HPO service')
     if len(sys.argv) == 1:
```

Comment:
I think this should check for (!REST) to see if the gRPC service needs to be started. This looks very flaky